packages/agoric-cli/test/agops-oracle-smoketest.sh

```diff
@@ -25,24 +28,27 @@ fi
 set -x
 
 # this is in economy-template.json in the oracleAddresses list (agoric1dy0yegdsev4xvce3dx7zrz2ad9pesf5svzud6y)
-# to use it run `agd keys oracle2 --interactive` and enter this mnenomic:
+# to use it run `agd keys --keyring-backend=test add oracle2 --interactive` and enter this mnenomic:
 # dizzy scale gentle good play scene certain acquire approve alarm retreat recycle inch journey fitness grass minimum learn funny way unlock what buzz upon
 WALLET2=oracle2
 
+WALLET_BECH32=$(agd keys show $WALLET --keyring-backend=test --output json | jq -r .address)
+WALLET2_BECH32=$(agd keys show $WALLET2 --keyring-backend=test --output json | jq -r .address)
+
 # Accept invitation to admin an oracle
 ORACLE_OFFER=$(mktemp -t agops.XXX)
 bin/agops oracle accept >|"$ORACLE_OFFER"
 jq ".body | fromjson" <"$ORACLE_OFFER"
-agoric wallet send --from "$WALLET" --offer "$ORACLE_OFFER"
+agoric wallet send --keyring-backend="test" --from "$WALLET_BECH32" --offer "$ORACLE_OFFER"
 # verify the offerId is readable from chain history
-agoric wallet show --from "$WALLET"
+agoric wallet show --keyring-backend="test" --from "$WALLET"
```

Comment:
So `show` works with the wallet name but not `send`? Interesting.

Reply:
Oh maybe it does now since https://github.com/Agoric/agoric-sdk/commit/b5d37ce5c251b8e7e2126f295b4d7dc61069d037, will try

Reply:
Yep `keyring-backend=test` works with `send` now from your changes, thanks!